flume/neore/scripts/package.sh

```diff
@@ -70,6 +70,7 @@ function download_flume(){
 	_log "#### Cleaning the temporal folders... ####"
 	rm -rf ${RPM_SOURCE_DIR}/${FLUME_WO_TAR}
 	rm -rf ${FLUME_WO_TAR}/docs # erase flume documentation
+        rm ${FLUME_WO_TAR}/conf/flume-conf.properties.template # we will add our own templates
```

Comment:
Uff, my IDE shows everything OK... The problem is the orginal file was indented by using a single "big" tab that is equivalent to 2 tabs in my IDE (Netbeans).